aurora/aurora-war/src/main/java/org/silverpeas/looks/aurora/LookAuroraHelper.java

```diff
@@ -670,4 +663,47 @@ public FreeZone getFreeZone() {
     }
     return null;
   }
+
+  public AuroraSpaceHomePage getHomePage(String spaceId) {
+    setSpaceIdAndSubSpaceId(spaceId);
+    String currentSpaceId = getSubSpaceId();
+
+    // get main information of space
+    SpaceInstLight space = getOrganisationController().getSpaceInstLightById(currentSpaceId);
+    AuroraSpaceHomePage homepage = new AuroraSpaceHomePage(this, space);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "homepage". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1488)